### ISSUE TITLE
Wide images in Media Library look messy and can be scaled up to fit the thumb display height

### ISSUE THREAD

**Issue by joehoyle** [humanmade/altis-media#212]
Due to how we have this "no max width" on images, it looks pretty bad when you have very wide images. I don't know if this change from the WP Media Library change was at the request of @noeltock, or how we got to have this, but IMO it's pretty terrible.

<img width="1624" alt="Screen Shot 2022-01-27 at 16 06 59" src="https://user-images.githubusercontent.com/161683/151375030-ee1fe0c0-9c4d-434a-88b4-626ecb0819ae.png">

**Comment by roborourke** [comment]
Cheers Joe. That specific image in your screenshot does indeed look bad, we probably need a maximum width we would show, especially as this looks like it's blown up to match the height displayed.

I think the initial goal was to have something like masonry or like how flickr displays images.

The problem it aimed to solve was in cases where users had multiple versions of the same or similar image with different crops, they would otherwise all be shown as a square thumbnail. I know Tachyon handles this but it can still happen easily, especially if using the inline cropper in the block editor.

Your example is a bit of an exception but agree we should handle it more nicely than that. I'll scope this issue a bit to handling very wide images.

**Comment by joehoyle** [comment]
Yeah I think the screenshot above is the worst-case, but would also like to at least be one voice that says that the masonry approach I don't think is a good one for productivity. Especially as we actually have a horizontal stack, so we're not able to use all the horizontal space.

IMO if we want to not show crops, we should have fixed size containers, and then we have the image resized to "max-width: 100pxl max-height: 100px" style.

**Comment by roborourke** [comment]
100x100 squares are a bit small to cram letterboxed images into so id probably go a bit bigger if we do this. Would be good to get some more voices in the mix.

**Comment by ferschubert-hm** [comment]
Improvements have been made in a previous PR, closing.
